drivers/hmis/app/jobs/hmis/create_fake_enrollments_job.rb

```diff
@@ -0,0 +1,311 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+require 'csv' # needed by FakeData
+
+# == Hmis::CreateFakeEnrollmentsJob
+#
+# Tool to generate fake Enrollment data for local development and staging environments.
+# It generates households of varying sizes. The default behavior is to generate open enrollments for existing clients.
+#
+# Records it generates:
+# - Clients (optionally)
+# - Enrollments (Household size varies from 1-4, except for NBN shelters which are set to 1)
+# - Exits (optionally)
+# - Disabilities (only for intake stage)
+# - Bed Night Services (for NBN projects, 1-100 per Enrollment)
+# - Intake and Exit Assessments
+#
+# Usage:
+#   Hmis::CreateFakeEnrollmentsJob.perform_now(num_households: 10)
+#   Hmis::CreateFakeEnrollmentsJob.perform_now(num_households: 10, generate_clients: true, exited: true, project_ids: [72])
+module Hmis
+  class CreateFakeEnrollmentsJob < BaseJob
+    include Hmis::Concerns::HmisArelHelper
+    include NotifierConfig
+    queue_as ENV.fetch('DJ_LONG_QUEUE_NAME', :long_running)
+
+    BATCH_SIZE = 5_000
+    HOUSEHOLD_SIZE_RANGE = 1..4
+    ENTRY_DATE_RANGE = 1..(365 * 3) # 1d-3y ago
+
+    def perform(
+      num_households:,
+      generate_clients: false,
+      exited: false,
+      project_ids: nil,
+      data_source_id: nil,
+      export_id: 'FAKED'
+    )
+      raise "won't create fake enrollments in production" if Rails.env.production?
+
+      setup_notifier('CreateFakeEnrollmentsJob')
+      @data_source = if data_source_id
+        ::GrdaWarehouse::DataSource.hmis.find(data_source_id)
+      else
+        ::GrdaWarehouse::DataSource.hmis.sole # 'sole' will raise if there are >1 HMIS data sources
+      end
```

Comment:
This will only ever be run manually, right? Maybe just pass the data source in directly to keep it simple?

Reply:
Yes but it's meant to be able to run in the background, eg `perform_later(data_source_id: x...)` - so I think for that it's better to pass the ID? Although I'm not sure, maybe it doesn't matter